runtime/js-compute-runtime/Makefile

```diff
@@ -67,6 +67,9 @@ else
   WASM_STRIP = wasm-opt --strip-debug -o $1 $1
```

Comment:
We likely do have to tackle this one head-on here I think - `wasm-opt` is in PATH in all our current workflows, so would need to be renamed. Binaryen bins are added to PATH in https://github.com/fastly/js-compute-runtime/blob/main/.github/workflows/main.yml#L93.